File: Assignment 01/Program 13/prog13.c
//Write a Python program to reverse the order of the items in the array.

#include<stdio.h>
#include<stdlib.h>

int main(){
    int n;
    printf("Enter the number of inputs: ");
    scanf("%d",&n);
    int *arr = (int*)malloc(n*sizeof(int));
    for (int i = 0; i < n; i++)
    {
        printf("Enter the number %d: ",i+1);
        scanf("%d",(arr+i));
    }
    int *arr1 = (int*)malloc(n*sizeof(int));
    for (int i = 0; i < n; i++)
    {
        arr1[i] = arr[n-i-1];
    }
    for (int i = 0; i < n; i++)
    {
        printf("%d ",arr1[i]);
    }
    free(arr);
    free(arr1);

    return 0;
}

File: Data Structure and Algorithm/Assignment 01/Program 13/prog13.py
#Write a Python program to reverse the order of the items in the array.

import array as arr

a1 = arr.array('i')
n = int(input("Enter the no. of inputs: "))
for i in range(n):
    val = int(input(f"Enter the value {i+1}: "))
    a1.append(val)

a2 = arr.array('i')
a2 = a1[::-1]
print(a2)